keyboards/gergo/keymaps/mim/keymap.c

```diff
@@ -0,0 +1,64 @@
+/* Copyright 2021 Martin Guilloux
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+enum combos {
+  FJ_ENTER,
+  G_INS_ENTER,
+  XCV_CAPSLOCK,
+};
+
+const uint16_t PROGMEM fj_combo[] = {KC_F, KC_J, COMBO_END};
+const uint16_t PROGMEM gins_combo[] = {KC_G, KC_INS, COMBO_END};
+const uint16_t PROGMEM xcv_combo[] = {KC_X, KC_C, KC_V, COMBO_END};
+
+combo_t key_combos[COMBO_COUNT] = {
+  [FJ_ENTER] = COMBO(fj_combo, KC_ENTER),
+  [G_INS_ENTER] = COMBO(gins_combo, KC_ENTER),
+  [XCV_CAPSLOCK] = COMBO(xcv_combo, KC_CAPSLOCK)
+};
+
+enum custom_keycodes {
+    TMUX = SAFE_RANGE,
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+    case TMUX:
+        if (record->event.pressed) {
+            SEND_STRING(SS_LCTL("x"));
```

Comment:
Why not just put `C(KC_X)` directly in the keymap?

Reply:
I recycled this snippet for longer SEND_STRING that isn't used anymore. fixed.